### PR TITLE
feat(efb): add EFB model

### PR DESCRIPTION
**Pull request by Benjozork**
Fixes #[issue_no]

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/panel/panel.cfg [hunk 1]
```diff
@@ -124,6 +124,13 @@ texture=$MFD2
 
 htmlgauge00=Airliners/A320_Neo/MFD/A320_Neo_MFD.html?Index=2, 0,0,1280,1280
 
+[VCockpit17]
+size_mm=1280,1280
```
Comment (ZigTag) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/panel/panel.cfg:128]: size is wrong

**Review by ZigTag — DISMISSED** [review]

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/panel/panel.cfg (hunk 1 above)
Comment (Benjozork) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/panel/panel.cfg:128]: It does not display anything

Comment (ZigTag) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/panel/panel.cfg:128]: We should at least have it displayed right so when we do try to get it to display something we have to change less

**Review by tyler58546 — APPROVED** [review]
LGTM

**Comment by FlightSimmer7700** [comment]
Category: QA Tester
Name: FlightSimmer7700#3468
Date of testing: 25/1/21
Version of the sim: 1.12.13.0
PR Tested: #3091 
Tier of testing: 1
Changes to observe:
EFB added

Testing technique: loaded in
Steps you've made to test the Pull Request:
1. Loaded in 
2. Checked model is present
3. Checked clickable spots

Flight notes (When done a full flight):

Media: Insert screenshots, recordings, or anything relevant to the review here.
Screenshot (455)

Issues (If Any): Clickable spots are off by a bit as shown by the screenshot above for the sides while the top and bottom are perfect
Steps to achieve the issue

Overall Rating: To be improved
Conclusions of the testing: Good overall just the clickable area that needs to be fixed.

**Comment by stescouse** [comment]
Category: QA Tester
Name: Discord User stescouse#7271
Date of testing: 25.1.2021
Version of the sim: 1.12.13.0
PR Tested: #3091
Tier of testing: 1
Changes to observe: EFB model added.

Testing technique: TryOut
Steps you've made to test the Pull Request:

1. Spawned C&D at SKBO
2. Checked EFB model present.
3. Checked clickable areas.

efb

Flight notes (If required: none

Issues (If Any): None


Overall Rating: Good
Conclusions of the testing: EFB model present as outlined in PR.

**Comment by ChruutvoLuzi** [comment]
Category: QA Tester
Name: ChruutvoLuzi | SD-PB.ch#8902
Date of testing: 24.1.2021
Version of the sim: 1.12.13.0
PR Tested: #3091 
Tier of testing: 1
Changes to observe: EFB Model

Testing technique: TryOut
Steps you've made to test the Pull Request:
1. Looked at the Aircraft in the Hangar
2. Spawned the Aircraft at LSZH Gate A44
3. Checked the Model

Flight notes (When done a full flight): n/a

Media:
EFB in Hangar
Microsoft Flight Simulator Screenshot 2021 01 25 - 21 25 51 39
EFB at LSZH
Microsoft Flight Simulator Screenshot 2021 01 25 - 21 29 24 25


Issues (If Any): The EFB Screen is a bit too dirty at certain light scenarios (especially hangar)
Steps to achieve the issue
Look at the efb in the hangar

Overall Rating: Almost Good
Conclusions of the testing: You could excuse the Textures for now by saying that it has not been used by the pilots yet so no need to clean it by them. Otherwise, if that was my Tablet, i would have flipped out by now. But that will be in the hands of the maintainers

**Comment by Awemeter** [comment]
Category: QA Tester
Name: Awemeter#7919 
Date of testing: 1/25/2021
Version of the sim: 1.12.13.0
PR Tested: #3091 
Tier of testing: 1
Changes to observe:
Add efb model

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. Spawned C&D at katl and started plane.
2. Looked to my left to check if efb model was there.
3. Looked around the plane looking for any model errors.


Flight notes (When done a full flight):

Media:

Issues (If Any): 


Overall Rating: Good
Conclusions of the testing: Epic Pr just one step closer to having the efb ready. But I did notice that the EFB was smaller then the holder in the plane.

**Comment by Lazarus320** [comment]
Category: QA Tester/Trial
Name: Lazarus#4286  
Date of testing: 26/1/2021
Version of the sim: 1.12.13.0
PR Tested: #3091 
Tier of testing: 1 
Changes to observe: EFB model presence 
 
Testing technique: Loaded c/d and tested the presence of the model.
Steps you've made to test the Pull Request:
1. Check that the EFB model is added. 
 
Flight notes: N/A

Media: 

EFB model 
Microsoft Flight Simulator Screenshot 2021 01 26 - 13 49 45 58

EFB clickable areas 
ezgif com-gif-maker
   

Issues (If Any): N/A
 
Overall Rating: Good. 
Conclusions of the testing: Looks great can't wait for more.

**Comment by HugoTro** [comment]
Category: QA Tester
Name: Hugo #3307
Date of testing: Current date 26/01/2021
Version of the sim: 1.12.13.0
PR Tested: #3091 
Tier of testing: 1
Changes to observe: EFB model

Testing technique: Look-at
Steps you've made to test the Pull Request:
1. Checked EFB after Aircart C&D


Media: [https://youtu.be/rGjE00Z_33g](url)

Issues : Apart from clickable areas, everything is good

Overall Rating: To be slightly improved
Conclusions of the testing: Good but clickable areas to fix, see media (maybe in another PR)

**Comment by JolteonYellow** [comment]
Category: QA Trial
Name: JolteonYellow#73278
Date of testing: 27/01/2021
Version of the sim: 1.12.13.0
PR Tested: #3091
Tier of testing: 1
Changes to observe:
EFB Added

Testing technique: Loaded In At Parking, EGLL
Steps you've made to test the Pull Request:
1. Loaded into parking 228 at EGLL
2. Checked the EFB is present
3. Checked clickable area

Flight notes (When done a full flight): N/A

Media: Insert screenshots, recordings, or anything relevant to the review here.
EFB1
EFB2

Issues (If Any): The clickable area is a bit too far to the left-hand side, 
Steps to achieve the issue


Overall Rating: Small improvement necessary.
Conclusions of the testing: EFB Present, the clickable area needs to be moved to the right, and the tablet is a bit too small for the holder height-wise. Also, the screen is a bit too dirty, especially in bright conditions as seen in image 2.

**Review by ZigTag — APPROVED** [review]
LGTM

**Pull request merged**